ansible/examples/slave_static.yml

```diff
@@ -301,6 +301,7 @@
       template:
         src: "templates/systemd/jenkins.service.j2"
         dest: "/etc/systemd/system/jenkins.service"
+        force: yes
```

Comment:
I thought this was for slave.jar not the systemd unit. Does this file need to change if the slave.jar gets updated?

Reply:
Whoops you right.